catalog/tests/dags/providers/provider_api_scripts/test_wikimedia_commons.py

```diff
@@ -399,6 +399,24 @@ def test_extract_creator_info_handles_link_as_partial_text(wmc):
     assert expect_creator_url == actual_creator_url
 
 
+@pytest.mark.parametrize(
+    "url, media_type, expected",
+    [
+        ("https://example.com/image.jpg", "image", "jpg"),
+        ("https://example.com/image.JpeG", "image", "jpeg"),
+        ("https://example.com/image.Png", "image", "png"),
+        ("https://example.com/image.GIF", "image", "gif"),
+        ("https://example.com/image.ogv", "image", None),
+        ("https://example.com/image.xyz", "image", None),
+        ("https://example.com/audio.mp3", "audio", "mp3"),
+        ("https://example.com/audio.ogg", "audio", "ogg"),
+        ("https://example.com/audio.WAV", "audio", "wav"),
+    ],
```

Comment:
Nit: do you mind adding comments to separate these?
```suggestion
    [
        # Valid images
        ("https://example.com/image.jpg", "image", "jpg"),
        ("https://example.com/image.JpeG", "image", "jpeg"),
        ("https://example.com/image.Png", "image", "png"),
        ("https://example.com/image.GIF", "image", "gif"),
        # Invalid (for our sake) images
        ("https://example.com/image.ogv", "image", None),
        ("https://example.com/image.xyz", "image", None),
        # Valid audio
        ("https://example.com/audio.mp3", "audio", "mp3"),
        ("https://example.com/audio.ogg", "audio", "ogg"),
        ("https://example.com/audio.WAV", "audio", "wav"),
    ],
```

Reply:
Good notes 👍